common/scala/src/main/scala/whisk/core/entity/Size.scala

```diff
@@ -69,6 +69,23 @@ case class ByteSize(size: Long, unit: SizeUnits.Unit) extends Ordered[ByteSize]
     ByteSize(commonSize, commonUnit)
   }
 
+  def *(other: Int): ByteSize = {
+    ByteSize(toBytes * other, SizeUnits.BYTE)
+  }
+
+  def /(other: ByteSize): Double = {
+    // Without throwing the exception the result would be `Infinity` here
```

Comment:
we could consider making this return a Try instead.

Reply:
What are the reasons for using a Try here?
On dividing `Int`s, you also get the response directly, instead of a `Try`, don't you?